src/Libraries/Microsoft.Extensions.AI.AzureAIInference/AzureAIInferenceChatClient.cs

```diff
@@ -16,12 +16,15 @@
 
 #pragma warning disable S1135 // Track uses of "TODO" tags
 #pragma warning disable S3011 // Reflection should not be used to increase accessibility of classes, methods, or fields
+#pragma warning disable SA1204 // Static elements should appear before instance elements
 
 namespace Microsoft.Extensions.AI;
 
 /// <summary>An <see cref="IChatClient"/> for an Azure AI Inference <see cref="ChatCompletionsClient"/>.</summary>
 public sealed partial class AzureAIInferenceChatClient : IChatClient
 {
+    private static readonly JsonElement _defaultParameterSchema = JsonDocument.Parse("{}").RootElement;
```

Comment:
Dispose of the JsonDocument? Or use `JsonSerializer.Deserialize<JsonElement>`?

Reply:
Related to https://github.com/dotnet/runtime/issues/108930. Given that it's a singleton I figured never returning this buffer to the pool is probably acceptable.

Reply:
This is also a temporary measure depending on whether we decide to move the schema generator to `Abstractions` making it available to the client implemetation.